assets/dev/js/editor/elements/views/container.js

```diff
@@ -141,6 +141,12 @@ const ContainerView = BaseElementView.extend( {
 		return parent.view.getNestingLevel() + 1;
 	},
 
+	checkIfParentIsWidget() {
+		const parent = this.container.parent;
+
+		return 'widget' === parent.type;
+	},
```

Comment:
@ArmanPurtoian In onRender(), can you wrap the `.set()` in an if statement?
`if ( ! this.model.get( 'isInner' ) ) {...}`
So that if we already set `isInner = true` previously then it wont run this logic all the time? WDYT?

Reply:
@ArmanPurtoian This `checkIfParentIsWidget()` is not clear enough. We won't understand the use case if we read this code months from now. I would rename the function name and also apply a different logic:

```
import { isWidgetSupportNesting } from 'elementor/modules/nested-elements/assets/js/editor/utils';

isNestedElementContentContainer() {
   const widgetType = this.container.parent.model.get('widgetType'); // 'nested-accordion'
   
   return widgetType && widgetType.trim() !== '' && isWidgetSupportNesting(widgetType);
}
```

This is just a guideline. Please investigate and let me know what you think?